src/services/codefixes/fixUnusedIdentifier.ts

```diff
@@ -171,6 +171,10 @@ namespace ts.codefix {
                 }
                 break;
 
+            case SyntaxKind.BindingElement:

```

Comment:
ArrayBindingPatterens are positional, so not sure how that works with them.. can you add a test for:
```ts
var [x, y, z] = [0, 1, 2];
y;
```